modules/migrations/github.go

```diff
@@ -74,7 +75,7 @@ type GithubDownloaderV3 struct {
 }
 
 // NewGithubDownloaderV3 creates a github Downloader via github v3 API
-func NewGithubDownloaderV3(ctx context.Context, userName, password, token, repoOwner, repoName string) *GithubDownloaderV3 {
+func NewGithubDownloaderV3(ctx context.Context, baseURL, userName, password, token, repoOwner, repoName string) *GithubDownloaderV3 {
```

Comment:
I think we could return with an error, so that we could know the error from `NewEnterpriseClient`

Reply:
The only error that `NewEnterpriseClient` throws is when parsing the given URLs, which have already been parsed prior to this func being called.

Reply:
@jolheiser ok then just "resolve" my comments below

Reply:
So we assume `NewEnterpriseClient` will only return the parse error.

Reply:
Right, and it never should because we've already asserted the URL is parsable prior to calling this.

I can handle the error if it would make us more comfortable for future maintaining, I just left it out since we shouldn't run in to it.